GoogleForCreators/web-stories-wp
# Improve page navigation performance

## Summary
This PR improves the page navigation speed by rendering an image for the active page when just navigating.
Previously, all the elements were displayed for an active page.

Further steps that would improve navigation performance as well (but are not directly related to navigation) could be determined in #8255


## To-do



## User-facing changes



## Testing Instructions






This PR can be tested by following these steps:

1. Make sure the "enable thumbnail caching" experiment is activated.
1. Add a few pages with some elements.
2. Verify that if you navigate to a new pages, an image is displayed as a thumbnail for that page (check the thumbnail source to verify that).
3. Verify that once you change something on the page, it's also reflected on the thumbnail.

## Checklist




---



Fixes #1916
